pkg/bootstrapper/bootstrapper_test.go

```diff
@@ -569,3 +569,71 @@ func TestPodManifestDirCreation(t *testing.T) {
 	assert.NoError(t, err, "error initializing kubelet files")
 	assert.DirExists(t, podManifestDirectory, "pod manifest directory was not created")
 }
+
+// TestExtractEnvVarsFromSystemdUnit tests that we can extract environment variables from a systemd unit file
+func TestExtractEnvVarsFromSystemdUnit(t *testing.T) {
```

Comment:
You continue to follow this pattern of combining many tests into one. Please break this pattern. Look at some of the other tests for better patterns. You can use helpers and construct this better instead of inlining everything.

Reply:
This is a different pattern than I was doing in the e2e tests. Using table driven tests is a commonly recommended pattern when writing go unit tests. 
https://github.com/golang/go/wiki/TableDrivenTests
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

If we do not wish to use this pattern I can remove my use of it, but there is a difference in this and my previous PRs for the WSU, which I agree was not the correct way to do things

Reply:
Thanks for the links as they explain your approach and I am not going to disagree with Dave Chaney :-) But yes, it was me trying to review the code at the airport and quickly thinking you were going with your old pattern. I withdraw my request for change.